lib/util.js

```diff
@@ -222,3 +222,22 @@ exports.parseFormats = function(info, debug) {
   formats.sort(exports.sortFormats);
   return formats;
 };
+
+
+
+/**
+ * @param {String} videoDescriptionHtml
+ * @return {String}
+ */
+exports.cleanVideoDescription = function(videoDescriptionHtml) {
+	if (!videoDescriptionHtml || videoDescriptionHtml == null) 
+		return null;
+
+	return entities.decode(videoDescriptionHtml
+		.replace(/\n/g, ' ')
+		.replace(/\s*\<\s*br\s*\/?\s*>\s*/gi, '\n')
+		.replace(/<\s*\/\s*p\s*>\s*<\s*p[^>]*>/gi, '\n')
+		.replace(/<.*?>/gi, '')
+	).trim();
```

Comment:
Can you change these tabs to 2 spaces instead for consistency with the rest of the project?